src/components/navigation/NavBar.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as PropTypes from 'prop-types';
+import { Logo } from '../logo/Logo';
+import styles from '../../styles/navigation/NavBar.scss';
+import DropDown from './GenerateDropdownItems';
+
+const LogoCointainer = () => (
+	<Link to="/" className='brand-logo left'>
+		<div className={`left ${styles.logo}`}>
+			<Logo />
+		</div>
+		<div className={`${styles.title} right hide-on-small-and-down`}>Authors Haven</div>
+	</Link>
+);
+
+class LoggedInNav extends React.Component {
+
+	render() {
+		return (
+			<ul className="right ">
+				<DropDown />
+
+				<li className={`hide-on-med-and-up ${styles.dropdown}`}>
+					<Link to="/">
+						<i className="material-icons">more_vert</i>
+					</Link>
+				</li>
+			</ul>
+		);
+	}
+}
+
+const NotLoggedInNav = () => (
+	<div className="right" style={{marginRight: '3em'}}>
+		<a href='#' className={styles.signup}>Signup</a>
+		<a href='#' className={styles.login}>Login</a>
+	</div>
+);
+
+const NavBar = (props) => (
+	<div className={`navbar-fixed ${styles['navbar-fixed']}`}>
+		<nav className={`navbar-fixed ${styles.white} ${styles.navigation}`}>
```

Comment:
for multiple class names, I would recommend using the npm package [classnames](https://www.npmjs.com/package/classnames)

Reply:
I have refactored the code with `npm` package `classnames`